GruntFile.js

```diff
@@ -340,7 +340,7 @@ module.exports = function (grunt) {
    //
 
    // Default task - compile & test
-   grunt.registerTask('default', ['tslint:src', 'compile', 'tests', 'visual']);
+   grunt.registerTask('default', ['compile', 'tests', 'visual']);
```

Comment:
Wouldn't we still want this? We still want to lint before compiling. Prettier will only do formatting but there could still be other errors.

Reply:
Sorry, suppose i got you wrong.
But which task you meant to remove there:  "3. Remove lint step from grunt"? 
This one was only linter task i found in grunt conf.

Reply:
Yep. After surfing for i while, i found this: https://github.com/prettier/prettier/issues/2447

Looks like configs are unextendable and they are not going to add this feature.  So, i just configured prettier the same as linter.